src/main/kotlin/com/autonomousapps/extension/Issue.kt

```diff
@@ -63,6 +64,18 @@ open class Issue @Inject constructor(objects: ObjectFactory) {
     excludes.addAll(ignore.toSet())
   }
 
+  /**
+   * All provided elements will be filtered out of the final advice. For example:
+   * ```
+   * exclude(libs.example, libs.some.thing)
+   * ```
+   * tells the plugin to exclude those dependencies in the final advice.
+   */
+  fun exclude(vararg ignore: Provider<MinimalExternalModuleDependency>) {
```

Comment:
could we have this call the existing `exclude()` method just above instead? Might make maintenance slightly easier if we ever have to change the above method.

Reply:
Done!